src/Primitive/Container.purs

```diff
@@ -140,6 +155,23 @@ component render =
           other -> a <$ do
             H.liftAff $ log $ show other
 
+      Mouse ms a -> do
+        st <- H.get
+        if not st.open then pure a else a <$ case ms of
+          Down -> do
+            H.liftAff $ log $ "mouse: down"
+            H.modify (_ { mouseDown = true })
+          Up -> do
+            H.liftAff $ log $ "mouse: up"
+            H.modify (_ { mouseDown = false })
+
+      Blur a -> do
```

Comment:
@whoadave How would this work if we want to close the menu after a single selection? Right now, it keeps the menu open even if you click in on a child item. If the user wanted to instead have the menu close after any selection is made, what would that look like on their end (with only access to the render function)?

Reply:
The parent would have to send the action back down to the child, I added the following to line 121 of the example `Container.purs` to get that to work:

`_ <- H.query unit $ H.action $ Container.Visibility Container.Off`